message_coder.cpp

```diff
@@ -54,78 +50,62 @@ void MessageCoder::slotDataAvailable()
         QByteArray content;
         quint16 checksum;
     } msg;
-    
-    // An important note about the transaction mechanism:
-    // Rollback ends a transaction and resets the stream position to the start of the transaction so it can be
```

Comment:
Is this logic implemented? If yes, make sure that useful descriptive comments are included.